code/modules/admin/admin_verbs.dm

```diff
@@ -957,14 +957,18 @@ var/list/admin_verbs_mod = list(
 		to_chat(usr, "player list is empty!")
 		return
 
-	var/mob/winner = input("Who's a winner?", "Achievement Winner") in player_list
+	var/mob/winner = input("Who's a winner?", "Achievement Winner", null) as null|anything in player_list

```

Comment:
Thats because those are null|anything without the in list, because there's no list obviously